homeassistant/components/vicare/climate.py

```diff
@@ -160,11 +160,15 @@ async def async_update(self) -> None:
         try:
             _room_temperature = None
             with suppress(PyViCareNotSupportedFeatureError):
-                _room_temperature = self._circuit.getRoomTemperature()
+                _room_temperature = await self.hass.async_add_executor_job(
```

Comment:
Just change the `async_update` method to a normal method called `update` instead. It's expensive to switch context to the executor and since the library is sync there's no reason to use async context here.

Reply:
👍 thanks for the recommendation, I reverted back to `update`.